package.json

```diff
@@ -1,26 +1,42 @@
 {
-  "name": "wikibase-rest-fantasies",
+  "name": "wikibase-rest-api-proposal",
   "version": "0.0.1",
-  "description": "Wikibase REST fantasies",
+  "description": "Wikibase REST API proposal",
   "main": "openapi.json",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "swagger-cli validate openapi.json",
-    "build": "swagger-cli bundle openapi.json -o docs/openapi.json",
-    "deploy": "gh-pages -d docs"
+    "lint": "swagger-cli validate specs/openapi.json",
+    "build": "run-s build:docs build:spec",
+    "build:docs": "webpack --config docs/webpack.config.js",
+    "build:spec": "swagger-cli bundle specs/openapi.json -o dist/openapi.json",
+    "watch": "webpack-dev-server --config docs/webpack.config.js",
+    "deploy": "gh-pages -d dist"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/wmde/wikibase-rest-fantasies.git"
+    "url": "git+https://github.com/wmde/wikibase-rest-api-proposal.git"
   },
   "author": "",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/wmde/wikibase-rest-fantasies/issues"
+    "url": "https://github.com/wmde/wikibase-rest-api-proposal/issues"
   },
-  "homepage": "https://github.com/wmde/wikibase-rest-fantasies#readme",
+  "homepage": "https://github.com/wmde/wikibase-rest-api-proposal#readme",
   "devDependencies": {
```

Comment:
As someone who does not have node/npm installed locally, seeing the build step become such an integral part of the project makes me crave for a reproducible environment to run it in.
I'll add something to the README.

Reply:
Created a suggestion at #43